lib/runnable.js

```diff
@@ -24,8 +24,10 @@ var MAX_TIMEOUT = Math.pow(2, 31) - 1;
 
 module.exports = Runnable;
 
+// Add "Additional properties" doc comment for hosted docs :)
```

Comment:
[Question] Was this intentionally left in?

Reply:
Yes, but happy to remove. Just thought it might be a bit unclear why I was writing "Look at Runnable source" in the Runnable source itself. To me at least it wasn't immediately obvious that these comments go directly to mochajs.org/api, figured it would help clarify :)

Reply:
Ahh, I think I see the confusion

```suggestion
// "Additional properties" doc comment added for hosted docs (mochajs.org/api)
```